Comparator.dcm

```diff
@@ -2,73 +2,73 @@ EESchema-DOCLIB  Version 2.0
 #
 $CMP AD8561
 D Ultrafast 7 ns Single Supply Comparator, DIP-8/SOIC-8/TSSOP-8
-K comparator ultrafast
+K cmp complementary push-pull
 F https://www.analog.com/media/en/technical-documentation/data-sheets/ad8561.pdf
 $ENDCMP
 #
 $CMP ADCMP350
 D Single-input comparator, internal 0.6V reference, Active-Low Open-Drain output, KS-4
-K single comparator with reference
+K cmp collector
 F https://www.analog.com/media/en/technical-documentation/data-sheets/ADCMP350_354_356.pdf
 $ENDCMP
 #
 $CMP ADCMP354
 D Single-input comparator, internal 0.6V reference, Active-High Open-Drain output, KS-4
-K single comparator with reference
+K cmp collector
 F https://www.analog.com/media/en/technical-documentation/data-sheets/ADCMP350_354_356.pdf
 $ENDCMP
 #
 $CMP ADCMP356
 D Single-input comparator, internal 0.6V reference, Active-High Push-Pull output, KS-4
-K single comparator with reference
+K cmp
 F https://www.analog.com/media/en/technical-documentation/data-sheets/ADCMP350_354_356.pdf
 $ENDCMP
 #
 $CMP LM2901
 D Quad Differential Comparators, DIP-14/SOIC-14/TSSOP-14
-K quad comparator
+K cmp open collector
```

Comment:
I think the quad keyword is usefull. Why did you remove it? There are similar occurrences below with 'dual'.

Reply:
'quad' already in description